invenio_drafts_resources/services/search.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+
+"""Search/Indexer module."""
+
+from invenio_records.api import RecordBase
+
+from ..drafts import DraftBase
+
+
+def draft_record_to_index(record):
+    """Return the index alias for records and drafts based objects.
+
+    Note: document type is deprecated and therefore returned as None
+    """
+    if isinstance(record, DraftBase):
+        return 'drafts', None
+    if isinstance(record, RecordBase):
+        return 'records', None
+    else:
+        return None, None
```

Comment:
Given the discussion about `'schemas'` yesterday, this would have to be changed to look for the `'schemas'` field right? The underlying problem being that there can be multiple versions of the draft schema as we evolve in time. 

I am fine with keeping this as is and investigating in another task.

Reply:
Shelving it: got numerous feedback from lars today. Will create an issue (working on it now) with all the points to change.